### PR TITLE
[candi] added force libseccomp2 installation while containerd install/update

### PR DESCRIPTION
**Pull request by duckhawk**
## Description
Added libseccomp2 installation if necessary while containerd install/update (details in issue https://github.com/containerd/containerd/discussions/6577)

## Why do we need it, and what problem does it solve?
Workaround for case from issue https://github.com/containerd/containerd/discussions/6577

## What is the expected result?

Correct containerd installation/update

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: candi
type: fix
summary: Added force libseccomp2 installation while containerd install/update (due to issue https://github.com/containerd/containerd/discussions/6577)
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'e2e/run/azure'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'e2e/run/gcp'...

**Comment by duckhawk** [comment]
/e2e/abort vsphere;Standard;containerd;1.23 3884660946 3884660946-1-con-1-23 /sys/deckhouse-oss/install:pr3504 ubuntu@10.220.203.26

**Comment by duckhawk** [comment]
/e2e/abort gcp;WithoutNAT;containerd;1.23 3884662773 3884662773-1-con-1-23 /sys/deckhouse-oss/install:pr3504 user@34.159.68.184

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'e2e/run/azure'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'e2e/run/gcp'...

File: candi/bashible/bundles/debian/node-group/031_install_containerd.sh.tpl [hunk 1]
```diff
@@ -90,6 +90,11 @@ containerd_tag="{{- index $.images.registrypackages (printf "containerdDebian%sB
   fi
 {{- end }}
 
+  # we need to force libseccomp2 install due issue https://github.com/containerd/containerd/discussions/6577
+  if [ $(dpkg-query -W -f='${Status}' libseccomp2 2>/dev/null | grep -c "ok installed") -eq 0 ];
+  then
+    bb-apt-install libseccomp2
+  fi
```
Comment (RomanenkoDenys) [inline candi/bashible/bundles/debian/node-group/031_install_containerd.sh.tpl:97]:
```suggestion
  bb-apt-install libseccomp2 --force
```

File: candi/bashible/bundles/ubuntu-lts/node-group/031_install_containerd.sh.tpl [hunk 2]
```diff
@@ -85,6 +85,11 @@ if [[ "$should_install_containerd" == true ]]; then
   fi
 {{- end }}
 
+  # we need to force libseccomp2 install due issue https://github.com/containerd/containerd/discussions/6577
+  if [ $(dpkg-query -W -f='${Status}' libseccomp2 2>/dev/null | grep -c "ok installed") -eq 0 ];
+  then
+    bb-apt-install libseccomp2
+  fi
```
Comment (RomanenkoDenys) [inline candi/bashible/bundles/ubuntu-lts/node-group/031_install_containerd.sh.tpl:92]: same

**Review by RomanenkoDenys — CHANGES_REQUESTED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'e2e/run/static'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'e2e/run/azure'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'e2e/run/aws'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'e2e/run/openstack'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'e2e/run/vsphere'...

**Comment by apolovov** [comment]
/e2e/abort openstack;Standard;containerd;1.23 3934089161 3934089161-1-con-1-23 /sys/deckhouse-oss/install:pr3504 debian@5.159.103.163
/e2e/abort vsphere;Standard;containerd;1.23 3934089567 3934089567-1-con-1-23 /sys/deckhouse-oss/install:pr3504 ubuntu@10.220.203.26

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by apolovov** [comment]
/e2e/abort vsphere;Standard;containerd;1.23 3934089567 3934089567-1-con-1-23 /sys/deckhouse-oss/install:pr3504 ubuntu@10.220.203.26

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've recognized your '/e2e/abort' command and started the workflow...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'e2e/run/vsphere'...

**Review by apolovov — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [3562](https://github.com/deckhouse/deckhouse/pull/3562) to the branch [release-1.42](https://github.com/deckhouse/deckhouse/tree/release-1.42) successful!

**Pull request merged**